src/superface-test.ts

```diff
@@ -165,26 +180,39 @@ export class SuperfaceTest {
     throw new UnexpectedError('Unexpected result object');
   }
 
-  private async replaceUnsupportedRecording(): Promise<void> {
-    const pathToUnsupported = this.composeRecordingPath('unsupported');
+  // static async collectData(): Promise<void> {
+  //   await collect();
+  // }
+
+  static async report(
```

Comment:
Because this is meant to be used by end user. I didn't want to export `report(...)`. So far we don't use anything except `SuperfaceTest` class and interfaces for it. Do you think it would be better to export `report()` function for end user?

Reply:
Ok, I get it. In code it looks kinda weird but let's not change it now :)